bundle_pyinstaller.sh

```diff
@@ -0,0 +1,194 @@
+}
+
+clone_repos() {
+    status="clone repositories"
+    echo "${cc_green}Status: $status...${cc_normal}"
+    set -x  # show commands
+
+    src="https://leap.se/git"
+    cd $REPOS_ROOT
+
+    for repo in $REPOSITORIES; do
+        echo "${cc_yellow}Status: cloning: $repo...${cc_normal}"
+        [ ! -d $repo ] && git clone -b develop $src/$repo --depth 1
+    done
+
+    set +x
+    echo "${cc_green}Status: $status done!${cc_normal}"
+}
+
+install_requirements() {
+    status="installing non-leap requirements"
+    echo "${cc_green}Status: $status...${cc_normal}"
+    set -x  # show commands
+    cd $REPOS_ROOT
+    source $VENV_DIR/bin/activate
+
+    for package in $PACKAGES; do
+        cd $REPOS_ROOT/$package
+        echo "${cc_yellow}Status: installing requirements for: $package...${cc_normal}"
+        ./pkg/pip_install_requirements.sh
+    done
+
+    # hack to solve gnupg version problem
+    pip uninstall -y gnupg && pip install gnupg
+
+    set +x
+    echo "${cc_green}Status: $status done.${cc_normal}"
+}
+
+create_sumo(){
+    echo "${cc_red}No sumo creation available in here.${cc_normal}"
+    exit  # no creation, get sumo from path
+    status="creating sumo tarball"
+    echo "${cc_green}Status: $status...${cc_normal}"
+    set -x  # show commands
+
+
+    # source $VENV_DIR/bin/activate
+    # cd $REPOS_ROOT/bitmask_client/
+    # python setup.py sdist --sumo
+}
+
+install_sumo(){
+    if [[ -z $1 ]]; then
+        echo "You need to specify a sumo tarball path parameter."
+        exit 1
+    fi
+
+    status="'installing' sumo tarball"
+    echo "${cc_green}Status: $status...${cc_normal}"
+    set -x  # show commands
+
+    source $VENV_DIR/bin/activate
+    SUMO_PATH=$1
+    # from http://stackoverflow.com/a/122340/687989
+    SITE_PACKAGES=`python -c "from distutils.sysconfig import get_python_lib; print(get_python_lib())"`
+
+    cd $SITE_PACKAGES
+
+    # the file name is like 'leap.bitmask-0.9.0rc1-12-g59bc704-SUMO.tar.gz'
+    # and the root path inside the archive for that file would be
+    # 'leap.bitmask-0.9.0rc1-12-g59bc704-SUMO'
+    base_path=`basename $SUMO_PATH .tar.gz`
+    tar xzf $SUMO_PATH $base_path/src/leap/ --strip-components=2
+}
+
+linux_tweaks(){
+    # - mv bitmask frozen binary to bitmask-app
+    # - cp bitmask-launcher (shell wrapper) to dist/bitmask/bitmask
+    # - cp all the needed helper files, tuf config etc
+    # - rename dist/bitmask distribution folder to bitmask-<release>-<date?>
+    exit
+}
+
+tweak_zope(){
+    # - create bare zope dir in site-packages
+    # - touch zope/__init__.py
+    # - move the zope.interface folder to zope/
+    # - move the zope.proxy folder to zope/
+    # - remove the zope* eggs in site-packages
+    # - remove any refs to these eggs in easy-install.pth ?? (unsure about this, talking from the top of my head) 
+
+    SITE_PACKAGES=`python -c "from distutils.sysconfig import get_python_lib; print(get_python_lib())"`
```

Comment:
I guess you can factor out this command with the above use and just execute in a subshell in here